lnst/Controller/RunSummaryFormatter.py

```diff
@@ -118,6 +117,6 @@ def format_run(self, run):
                 output_lines.extend(self._format_data(res.data))
 
         output_lines.append("Overall result of this Run: {}".
-                            format(self._format_success(run.overall_result)))
+                            format(run.overall_result))
```

Comment:
why remove the formatting/colorization here?

Reply:
sure